src/main/java/autoreconnect/mixin/DisconnectedScreensMixin.java

```diff
@@ -24,14 +28,15 @@ public DisconnectedScreensMixin(Text text) {
         super(text);
     }
 
+    public ButtonWidget.PressAction pressAction = btn -> AutoReconnect.schedule(() -> MinecraftClient.getInstance().execute(this::manualReconnect), 100, TimeUnit.MILLISECONDS);
```

Comment:
You should put the press action as a variable directly into the constructor, as it's only used there.